src/components/elements/CommonMenuButton.tsx

```diff
@@ -98,28 +99,30 @@ const CommonMenuButton = ({
         }}
         {...MenuProps}
       >
-        {items.map(({ key, to, title, divider, onClick, disabled }) =>
-          divider ? (
-            <Divider key={key} />
-          ) : to ? (
-            <MenuItem key={key} component={RouterLink} to={to}>
-              {title}
-            </MenuItem>
-          ) : (
-            <MenuItem
-              key={key}
-              onClick={() => {
-                if (onClick) {
-                  // close menu before triggering onClick
-                  setAnchorEl(null);
-                  onClick();
-                }
-              }}
-              disabled={disabled}
-            >
-              {title}
-            </MenuItem>
-          )
+        {items.map(
+          ({ key, to, title, divider, onClick, disabled, ariaLabel }) =>
+            divider ? (
+              <Divider key={key} />
+            ) : to ? (
+              <MenuItem key={key} component={RouterLink} to={to}>
+                {title}
+              </MenuItem>
+            ) : (
+              <MenuItem
+                key={key}
+                onClick={() => {
+                  if (onClick) {
+                    // close menu before triggering onClick
+                    setAnchorEl(null);
+                    onClick();
+                  }
+                }}
+                disabled={disabled}
+                aria-label={ariaLabel}
```

Comment:
> Every menuitem must have an accessible name. This name comes from the element's contents by default. If the contents don't provide for a useful accessible name, [aria-labelledby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby) can be used to reference visible label. If no visible content is available to provide the accessible name, an accessible name can be provided with [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label).

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menuitem_role

Based on that description, I'm not sure this is the best solution. In this case the menu items are "Edit" and "Delete", right? Were you having trouble selecting those just based on text (with capybara)?

Reply:
Hm, yes I was, but I can give it another try.

Reply:
Ok, yes, it is possible like this: https://github.com/greenriver/hmis-warehouse/pull/4627/commits/b1ccbc2d98eaa1448d694e52e430323ea0cac389

Reply:
Hmm ok, I think we want to avoid relying on MUI classes in the system tests as much as possible, since those can change, and are related to styling not function. Can you use `li[role="menuitem"]` instead?

Reply:
If not, I guess aria label is better than using MUI class

Reply:
It seems to work with `find('li', exact_text: 'Edit')`. (Not `find("li[exact_text='Edit']")` 😞 )